src/hooks/api/member/useApiSendVerifyEmail.ts

```diff
@@ -2,13 +2,15 @@ import { api } from "@/api";
 
 interface sendVerifyEmailProps {
   email: string;
+  emailPurpose: string;
```

Comment:
`string` 형태로 문자열에 대한 자율성을 주는 것보다 방식이 매개변수에 들어오는 문자열을 제한적으로 주는 것도 좋을 것 같습니다.

```suggestion
  emailPurpose: 'SIGNUP' | 'SUBSCRIBE'
```